optax/losses/_self_supervised.py

```diff
@@ -55,7 +86,8 @@ def ntxent(
   # cosine similarity matrix
   xcs = (
       _regression.cosine_similarity(
-          embeddings[None, :, :], embeddings[:, None, :]
+          embeddings[None, :, :], embeddings[:, None, :],
+          epsilon=np.finfo(embeddings.dtype).eps
```

Comment:
no need to import numpy, you can do the same with jnp instead of np

Reply:
Got it! Will change shortly.